src/glfw-sandbox/engine.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
```

Comment:
// Same tip with hpp and cpp applies here

Reply:
Honestly that popped up from the Visual Studio by default, we can use header guards as Nico explained :)

Reply:
No no, `#pragma once` is nice, my comment means I would expect to have `engine.hpp` and `engine.cpp` :)